src/Interpreters/InterpreterDropQuery.cpp

```diff
@@ -381,6 +384,29 @@ BlockIO InterpreterDropQuery::executeToDatabase(const ASTDropQuery & query)
     return res;
 }
 
+bool matchesLikeWithRE2(const String & haystack,
+                        const String & like_pattern,
+                        bool case_insensitive)
+{
+    /// Converts LIKE pattern (with % and _) to a RE2 pattern
+    String regex_str = likePatternToRegexp(like_pattern);
+
+    LOG_TRACE(getLogger("TRUNCATING"), "regex_str: {}", regex_str);
+
+    /// Sets up RE2 with case insensitivity if needed
+    RE2::Options options;
```

Comment:
I would also set `regexp_options.set_log_errors(false);` just in case to not write error messages to stdout.